### PR TITLE
[Content] Tutschku article

### PR DESCRIPTION
**Pull request by jdchart**

### CODE REVIEW

File: src/routes/madewithflucoma/tutschku.svx [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+title: NMF Piano Tracker for Symbolic Audio Effects
+artist: Hans Tutschku
+blurb: |
+    In this article, we’ll take a look at some of the work that Hans Tutschku did using the FluCoMa tools, notably the conception of a piano tracker that uses NMF and how this allows him to perform symbolic operations of audio.
+tags: 
+    - nmf
+    - hans tutschku
+    - mark knoof
+    - nmf match
```
Comment (jamesb93) [inline src/routes/madewithflucoma/tutschku.svx:10]: nmfmatch so it links up against the object

File: src/routes/madewithflucoma/tutschku.svx [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+title: NMF Piano Tracker for Symbolic Audio Effects
+artist: Hans Tutschku
+blurb: |
+    In this article, we’ll take a look at some of the work that Hans Tutschku did using the FluCoMa tools, notably the conception of a piano tracker that uses NMF and how this allows him to perform symbolic operations of audio.
+tags: 
+    - nmf
+    - hans tutschku
```
Comment (jamesb93) [inline src/routes/madewithflucoma/tutschku.svx:8]: this will get caught by the artist field so its superfluous

**Comment by jdchart** [comment]
very true, corrected

**Comment by jamesb93** [comment]
Let me know when you want review @jdchart.

**Comment by jdchart** [comment]
Yep ! It's been sent to Hans, if you have any comments they're welcome!

**Comment by jamesb93** [comment]
This will break beacuse svelte has made a breaking change upstream (which vercel uses to build against). If you are happy with how it looks locally @jdchart merge to your heart's content and I'll fix it after the fact. or you can run `npm update` and commit the `package.json` whic should work and merge just fine into `main`.

**Comment by jdchart** [comment]
@jamesb93 yeah i was wondering why even when i went onto the main branch and run locally the same tag error was coming up. It's fine for me and Hans liked it, but we'll wait till thursday i suppose !

**Pull request merged**